js/Assemble.js

```diff
@@ -0,0 +1,21 @@
+$(document).ready(function () {
+	$('#top').change(function () {
+		var piece = $(this).val();
+		$('#topPiece').attr('src', window.location.pathname.replace("/assemble", "") + '/images/bot/' + piece + '.jpeg');
+	});
+	$('#middle').change(function () {
+		var piece = $(this).val();
+		$('#middlePiece').attr('src', window.location.pathname.replace("/assemble", "") + '/images/bot/' + piece + '.jpeg');
+	});
+	$('#bottom').change(function () {
+		var piece = $(this).val();
+		$('#bottomPiece').attr('src', window.location.pathname.replace("/assemble", "") + '/images/bot/' + piece + '.jpeg');
+	});
+});
+
+$(document).ready(function () {
```

Comment:
You don't need another 
`$(document).ready(function() {...});`.
Just add it into the previous one.
Instructions:  **delete lines 14 - 16**